### PR TITLE
Update NERC-NEUTRONRCH7BT

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes https://github.com/betaflight/betaflight/issues/13052

### CODE REVIEW

**Review by nerdCopter — APPROVED** [review]

File: configs/default/NERC-NEUTRONRCH7BT.config [hunk 1]
```diff
@@ -3,9 +3,10 @@
 #define USE_ACC
 #define USE_GYRO
 #define USE_ACCGYRO_BMI270
-#define USE_FLASH
```
Comment (asizon) [inline configs/default/NERC-NEUTRONRCH7BT.config:6]: This FC doesnt have flash memory for BB??

Comment (nerdCopter) [reply configs/default/NERC-NEUTRONRCH7BT.config:6]: guess it does.  it is difficult find official NeutronRC sources. 
https://newbeedrone.com/products/neutronrc-h7bt

Comment (haslinghuis) [reply configs/default/NERC-NEUTRONRCH7BT.config:6]: Board uses SD NAND and SDCard interface like Speedybee F7 V3.
The W25M02G is unsupported and not recommended.

**Review by asizon — APPROVED** [review]

**Comment by MouseMike1** [comment]
Thank you! Everything works great. Problem solved.

**Pull request merged**